lisa/tools/tlb_stress.py

```diff
@@ -0,0 +1,1036 @@
+                    result.exit_code == 0
+                    and "<not supported>" not in result.stderr
+                    and "not supported" not in result.stderr
+                )
+                event_support[event] = supported
+                self._log.debug(
+                    f"Perf event {event}: "
+                    f"{'supported' if supported else 'not supported'}"
+                )
+            except Exception:
+                event_support[event] = False
+
+        return event_support
+
+    def _get_fallback_events(self, event_support: Dict[str, bool]) -> str:
+        """Get fallback event list based on what's supported"""
+        # Always include generic software events
+        base_events = []
+
+        # Add supported hardware events
+        if event_support.get("context-switches", False):
+            base_events.append("context-switches")
+        if event_support.get("cpu-migrations", False):
+            base_events.append("cpu-migrations")
+        if event_support.get("page-faults", False):
+            base_events.append("page-faults")
+
+        # Add PMU events if supported
+        if event_support.get("dTLB-load-misses", False):
+            base_events.append("dTLB-load-misses")
+        if event_support.get("iTLB-load-misses", False):
+            base_events.append("iTLB-load-misses")
+
+        # If no events supported, use minimal set
+        if not base_events:
+            base_events = ["context-switches", "page-faults"]
+            self._log.warning(
+                "No perf events detected as supported, using minimal fallback set"
+            )
+
+        return ",".join(base_events)
+
+    def _install_deps(self) -> None:
+        """Install build dependencies for supported distributions only"""
+        from lisa.operating_system import CBLMariner, Debian
+
+        if isinstance(self.node.os, Debian):
+            self._log.debug("Detected Debian/Ubuntu distribution")
+            self._install_debian_deps()
+        elif isinstance(self.node.os, CBLMariner):
+            self._log.debug("Detected Azure Linux/CBL Mariner distribution")
+            self._install_azurelinux_deps()
+        else:
+            # Clear error message for unsupported distributions
+            distro_info = getattr(self.node.os, "name", "Unknown")
+            raise RuntimeError(
+                f"Unsupported distribution: {distro_info}. "
+                "This tool only supports Ubuntu (Debian) and Azure Linux "
+            )
+
+    def _install_debian_deps(self) -> None:
+        """Install dependencies for Debian/Ubuntu systems"""
+        packages = [
+            "build-essential",
+            "libc6-dev",
+            "linux-headers-generic",
+            "stress-ng",
+            "numactl",
+        ]
+        try:
+            packages.append("linux-tools-generic")
+        except Exception:
+            pass
+
+        self._log.debug("Installing Debian/Ubuntu build dependencies...")
```

Comment:
The debug lines are unnecessary. LISA already logs each command and its exit status.

Reply:
done